src/js/controllers/chatController.js

```diff
@@ -153,7 +153,7 @@ module.exports = ViewController.extend({
                 .then(this._getConversation)
                 .then(this._initFaye)
                 .then(_.bind(function(conversation) {
-                    this.conversationInitiated = true;
+                    this.conversationInitiated = !conversation.isNew();
```

Comment:
That's why it was only working on reload... the conversation was always marked as initiated so it was not refetching from the server.